### PR TITLE
New package: bucklespring-1.4.0

### PR DESCRIPTION
**Pull request by abenson**

### CODE REVIEW

File: srcpkgs/bucklespring/template [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	vbin buckle
+
+	vmkdir usr/share/${pkgname}
+	vcopy wav usr/share/${pkgname}
+}
+
+bucklespring-data_package() {
+	short_desc+=" - data files"
+	noarch=yes
+
```
Comment (maxice8) [inline srcpkgs/bucklespring/template]: extra blank line

File: srcpkgs/bucklespring/template [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+depends="bucklespring-data"
+short_desc="Emulate the sound of the IBM Model-M while typing"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/zevv/bucklespring"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=b38b2c0ec905573a243cb9db543f8eeb92e4d040025ef338d8bcfcfd5ccd5348
+
+if [ $XBPS_CROSS_BASE ]; then
+	make_build_args+=" CROSS=$XBPS_CROSS_TRIPLET-"
```
Comment (maxice8) [inline srcpkgs/bucklespring/template]: isn't there an extra dash at the end or is it required ?

Comment (abenson) [reply srcpkgs/bucklespring/template]: Yeah, they do this stupid thing: 

~~~
CC 	?= $(CROSS)gcc
~~~

For example in their own makefile:

~~~
 CROSS   := i686-w64-mingw32-
~~~

Comment (abenson) [reply srcpkgs/bucklespring/template]: I guess I could just just do CC=$CC

Comment (abenson) [reply srcpkgs/bucklespring/template]: I thought $CC or $CC_FOR_BUILD would give me the right cross compiler?

**Pull request merged**